.github/workflows/WE2E_tests.yaml

```diff
@@ -0,0 +1,123 @@
+    if: ${{ github.event.label.name == 'ci-aws-intel-WE' }}
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+ 
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          clean: false
+      - id: set-matrix
+        run: | 
+          echo "::set-output name=matrix::$(cat ufs-srweather-app-${{ needs.get-run-id.outputs.run-id }}/regional_workflow/tests/WE2E/machine_suites/ci.txt | jq -R -s -c 'split("\n")[:-1]')"
+          echo 
+
+  create-experiment-directory:
+    needs: [ set-matrix ]
+    runs-on: self-hosted
+    if: ${{ github.event.label.name == 'ci-aws-intel-WE' }}
+    # We use THIS workflow's run id to set a new experiment directory, not the SRWA build workflow run_id 
+    steps:
+      - name: Create experiment directory 
+        run: |
+          mkdir -p /scratch1/ci-WE2E-test/${{ github.run_id }}
+
+  submit-all-tests:
+    name: Submit experiments to ./run_WE2E_tests.sh
+    needs: [ set-matrix, create-experiment-directory, get-run-id ]
+    runs-on: self-hosted
+    if: ${{ github.event.label.name == 'ci-aws-intel-WE' }}
+
+    steps:
+      - name: Submit all experiments (WE2E tests) 
+        run: | 
+          cd ufs-srweather-app-${{ needs.get-run-id.outputs.run-id }}/regional_workflow/tests/WE2E/
+          module use -a /scratch1/miniconda3/contrib_miniconda3/modulefiles
+          module load miniconda3/4.11.0
+          conda activate regional_workflow
+          ./run_WE2E_tests.sh \
+            tests_file=machine_suites/ci.txt \
+            machine=aws \
+            account=account \
+            expt_basedir=/scratch1/ci-WE2E-test/${{ github.run_id }} \
+            exec_subdir=bin_intel/bin \
+            cron_relaunch_intvl_mnts=01 \
+            build_mod_fn=build_aws_intel
+
+  check-overall-we2e-status:
+    name: Check all of the experiments were submitted
+    needs: [ set-matrix, create-experiment-directory, submit-all-tests, get-run-id ]
+    runs-on: self-hosted
+    if: ${{ github.event.label.name == 'ci-aws-intel-WE' }}
+    steps:
+      - name: Check the status of all experiments
+        run: |
+          echo "Check Workflow Generation/Status"
+          cd ufs-srweather-app-${{ needs.get-run-id.outputs.run-id }}/regional_workflow/tests/WE2E/
+          ./get_expts_status.sh expts_basedir=/scratch1/ci-WE2E-test/${{ github.run_id }}
+    
+  check-each-we2e-status: 
+    name: Check ${{ matrix.WE2Etest }} Status
+    needs: [ set-matrix, create-experiment-directory, submit-all-tests, check-overall-we2e-status, get-run-id ]
+    runs-on: self-hosted
+    if: ${{ github.event.label.name == 'ci-aws-intel-WE' }}
+    strategy:
+        fail-fast: false
+        matrix:
+            WE2Etest: ${{ fromJson(needs.set-matrix.outputs.matrix) }}
+
+    steps: 
+      - name: Check Workflow Status
+        run: |
+          cd /scratch1/ci-WE2E-test/${{ github.run_id }}/${{ matrix.WE2Etest }}
+          sleep 60
+          tail -40 log.launch_FV3LAM_wflow
```

Comment:
Any particular reason we need 40 lines of output here?

Reply:
40 has been the number I've used in the past to get a better gauge on where the workflow stands, and if the first couple of tasks were submitted. I can certainly reduce that number, if you don't think it is valuable to see whether rocoto has initiated any tasks.